### ISSUE TITLE
Message Viewer

### ISSUE THREAD

**Issue by nclm** [CopOnTheRun/FB-Message-Parser#6]
Great project!

A side tool could be a conversation viewer, reading the JSON. Opened in a browser, the messages would be browsable, separated by conversations/persons, searchable, etc. All the things the local archive messages html file doesn’t do but the online original message feature does.

**Comment by CopOnTheRun** [comment]
Yup, when I first conceived of the project I had planned to add many of the features you've suggested. Unfortunately I got busy with school, and other interests, so I haven't really followed up on those features, and I don't think I will for some time. However, I'd urge you to check out [jsharkey13's fork](https://github.com/jsharkey13/facebook_message_parser) which is much more comprehensive in its feature set!

**Comment by jsharkey13** [comment]
I tried making a simple proof of concept server interface, building on @CopOnTheRun 's original format (which made a great API) to allow viewing in the browser. Got a little carried away with the HTML/CSS/SCSS stuff; but the result is here: https://github.com/jsharkey13/flask_facebook_messages.  It might do something like what you want. This is apparently what happens when you finish university and start hanging out with computer scientists . . .

(PS - Sorry to hijack your issue!)
